# from django.core.exceptions import ValidationError


class HolidaysValidator:
    def __init__(self): ...

    def __call__(self):
        return True


class HolidaysimageValidator:
    def __init__(self): ...

    def __call__(self):
        return True
